packages/server/graphql/public/typeDefs/GcalIntegration.graphql

```diff
@@ -11,4 +11,9 @@ type GcalIntegration {
   The cloud provider the team member may choose to integrate with. Nullable based on env vars
   """
   cloudProvider: IntegrationProviderOAuth2
+
+  """
+  Events for specific time periods
+  """
+  events(startDate: DateTime!, endDate: DateTime!): [GcalIntegrationEvent!]
```

Comment:
I think generally yes - in cases where this object exists but isn't configured (i.e. before the user auths), returning `null` would reflect the absence of a list of events, rather than the presence of an empty list of events.

However, looking at other integrations, the pattern has been for this to be non-null, so I've changed this field to be a non-null array.